### PR TITLE
Allow to start a toplogy without FSC nodes

### PR DESCRIPTION
**Pull request by mbrandenburger**
This PR allows to run topologies even without FSC nodes, which is very helpful when developing nwo platforms and extensions.

### CODE REVIEW

**Review by adecaro — APPROVED** [review]
LGTM

File: integration/integration.go [hunk 1]
```diff
@@ -293,24 +293,28 @@ func (i *Infrastructure) initNWO() {
...
 		default:
+			logger.Infof("Register %s", label)
 			factory, ok := i.PlatformFactories[label]
 			Expect(ok).To(BeTrue(), "expected to find platform [%s]", label)
 			platforms = append(platforms, factory.New(i.Ctx, topology, i.BuildServer.Client()))
 		}
 	}
-	// Add FSC platform
-	fcsPlatform := i.PlatformFactories["fsc"].New(i.Ctx, fscTopology, i.BuildServer.Client()).(*fsc.Platform)
-	platforms = append(platforms, fcsPlatform)
```
Comment (alexandrosfilios) [inline integration/integration.go:306]: I think the point here was to have fscPlatform as the last one maybe

File: integration/integration.go [hunk 2]
```diff
@@ -293,24 +293,28 @@ func (i *Infrastructure) initNWO() {
 		switch label {
 		case "fsc":
 			// treat fsc as special
-			fscTopology = topology
-			continue
+			factory, ok := i.PlatformFactories["fsc"]
+			Expect(ok).To(BeTrue(), "expected to find platform [%s]", label)
```
Comment (alexandrosfilios) [inline integration/integration.go:297]: do we need this assertion? I think it should work without FSC factories (though I don't remember whether we have any such tests)

File: integration/integration.go (hunk 1 above)
Comment (mbrandenburger) [reply integration/integration.go:306]: ah good point

File: integration/integration.go (hunk 2 above)
Comment (mbrandenburger) [reply integration/integration.go:297]: true - a few lines before we register fsc factory in case it does not exist

**Pull request merged**